samples/v1.0/Elements/ColumnSet.Input.Text.VerticalStretch.json

```diff
@@ -0,0 +1,48 @@
+{
+	"$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+	"type": "AdaptiveCard",
```

Comment:
Since these are not v1.0 features, i don't think these should go in the v1.0 folder. We should come up with a plan for vnext samples, but in the meantime maybe put these in Test?